.github/workflows/trivy.yml

```diff
@@ -15,11 +15,11 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@main
 
       - name: Build an image from Dockerfile
         run: |
-          docker build -t spacelift:${{ github.sha }} .
+          docker build --build-arg TARGETARCH=amd64 -t spacelift:${{ github.sha }} .
```

Comment:
Random question (that I don't know the answer to) - are there likely to be different vulnerabilities for different platforms? If not then this is fine, but otherwise maybe we need to run Trivy against each arch?

Reply:
possibly! I added a matrix.